python/treelite/gtil/gtil.py

```diff
@@ -13,21 +13,29 @@
 from ..util import c_str
 
 
+@dataclass
 class GTILConfig:
     """Object holding configuration data"""
 
-    # pylint: disable=too-few-public-methods
+    predict_type: str
```

Comment:
You could be even more explicit here by using a `typing.Literal`:
```suggestion
    predict_type: Literal["default", "raw", "leaf_id", "score_per_tree"]
```

Reply:
Done